Python/optimizer.c

```diff
@@ -657,7 +652,15 @@ translate_bytecode_to_trace(
                                 oparg = offset;
                                 assert(uop == _SAVE_RETURN_OFFSET);
                                 break;
-
+                            case OPARG_REPLACED:
+                                uop = _PyUOp_Replacements[uop];
+                                assert(uop != 0);
+                                if (uop == _FOR_ITER_TIER_TWO) {
+                                    target += 1 + INLINE_CACHE_ENTRIES_FOR_ITER + oparg + 1;
+                                    assert(_PyCode_CODE(code)[target-1].op.code == END_FOR ||
+                                            _PyCode_CODE(code)[target-1].op.code == INSTRUMENTED_END_FOR);
```

Comment:
@markshannon, this assertion is now failing for me on `main` when running `./python -Xuops` (bisected to this PR).

Reply:
Pretty sure this has to do with how the code before this PR compensated for `extras` (adding it to or subtracting it from `oparg` depending on the opcode).

Reply:
https://github.com/python/cpython/pull/113394